File: GuardianMonsters/guardians/src/de/limbusdev/guardianmonsters/guardians/monsters/ISpeciesDescriptionService.java
package de.limbusdev.guardianmonsters.guardians.monsters;

import de.limbusdev.guardianmonsters.guardians.GuardiansServiceLocator;

/**
 * IGuardianDescriptionService
 *
 * @author Georg Eckert 2017
 */

public interface ISpeciesDescriptionService extends GuardiansServiceLocator.Service
{
    SpeciesDescription getSpeciesDescription(int speciesID);
    String getCommonNameById(int speciesID, int form);
}

File: GuardianMonsters/guardians/src/de/limbusdev/guardianmonsters/guardians/monsters/ISpeciesDescriptionService.kt
package de.limbusdev.guardianmonsters.guardians.monsters

import de.limbusdev.guardianmonsters.guardians.GuardiansServiceLocator

/**
 * IGuardianDescriptionService
 *
 * @author Georg Eckert 2017
 */

interface ISpeciesDescriptionService : GuardiansServiceLocator.Service
{
    fun getSpeciesDescription(speciesID: Int): SpeciesDescription
    fun getCommonNameById(speciesID: Int, form: Int): String
}
